ruby_snowflake_client.gemspec

```diff
@@ -24,4 +24,7 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency "bundler"
   s.add_development_dependency "rake"
+  s.add_development_dependency "rspec"
+  s.add_development_dependency "pry"
+  s.add_development_dependency "dotenv"
```

Comment:
Just a side note: I haven't been able to figure out any advantage to including these in the gemspec instead of just putting them into the Gemfile. And putting them in the Gemfile keeps the gemspec cleaner.  Any thoughts on that?